src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, operand) {
+      this.result = callback(operand, this.result);
+
+      return this;
+    },
+
+    add(operand, result) {
+      return result + operand;
+    },
+
+    subtract(operand, result) {
+      return result - operand;
```

Comment:
Use `result -= operand;` instead of `return result - operand;` for consistency and to follow the checklist advice.